src/AbstractView.php

```diff
@@ -119,91 +119,4 @@ public function add($object, $args = null): self
 
         return $object;
     }
-
-    // }}}
-
-    // {{{ Sticky URLs
-
-    /** @var string[] stickyGet arguments */
-    public $stickyArgs = [];
```

Comment:
So callback does no longer has sticky args?

Reply:
Yes, only View have stickyArgs

Reply:
Callback can be API that should preserve args. I am neutral on this change.

Reply:
Since sticky get is applied to the owner of the callback (View) I think it is ok to have stickyArgs property to View only.

I believe that Callback should not manage URL at all but only the arguments needed to run the callback function. It should be to the View using the callback to properly managed its URL. 

For example in Loader::jsLoad() method it should be:

````
$this->url($this->cb->getUrlArguments()),
````
instead of:
```
$this->cb->getUrl(),
````

Reply:
`$this->url($this->cb->getUrlArguments()),` I am fine with this, collecting trigger/sticky names is special case...